components/upload/chunk-upload.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Chunk Upload
+page_title: Chunk Upload
+description: Enable and configure chunk upload in Upload for Blazor.
+slug: upload-chunk
+tags: telerik,blazor,upload,chunk
+published: True
+position: 3
+---
+
+# Chunk Upload
+
+Chunk upload enables file uploads by dividing the file into smaller parts (chunks) and sending them in multiple requests to the remote endpoint, where they are reassembled into the final file.
```

Comment:
May be we can also list the benefit here and why one would need/want to enable the chunk upload.